tests/unit/components/initials-avatar-test.js

```diff
@@ -104,12 +103,13 @@ test('handle empty first name with company', function(assert) {
 test('displays initials if no image is given', function(assert) {
   assert.expect(2);
 
-  var component = this.subject({
+  let component = this.subject({
     firstName: "Barack",
     lastName: "Obama",
   });
 
-  var $component = this.append();
+  this.append();
```

Comment:
I know you weren't the one to write them, but these `append` tests should be integration tests instead.

```js
import { moduleForComponent, test } from 'ember-qunit';
import hbs from 'htmlbars-inline-precompile';

moduleForComponent('initials-avatar', 'Integration | Component | initials-avatar ', {
  integration: true
});

test('displays initials if no image is given', function(assert) {
  assert.expect(2);

  this.render(hbs`
    {{initials-avatar
      id="initials-avatar"
      firstName="Barack"
      lastName="Obama"
    }}
  `);

  assert.equal(this.$('#initials-avatar').attr('style'), undefined);
  assert.equal(this.$('#initials-avatar').text().trim(), 'BO');
});

test('displays image if one is provided', function(assert) {
  assert.expect(2);

  this.render(hbs`
    {{initials-avatar
      id="initials-avatar"
      image="http://example.com/potus.jpg"
    }}
  `);

  assert.equal(this.$('#initials-avatar').attr('style'), 'background-image: url(http://example.com/potus.jpg); background-size: cover');
  assert.equal(this.$('#initials-avatar').text().trim(), '');
});
```